pkg/collector/corechecks/sbom/processor.go

```diff
@@ -179,14 +179,57 @@ func (p *processor) processContainerImagesRefresh(allImages []*workloadmeta.Cont
 	}
 }
 
-func (p *processor) processHostRefresh() {
-	if !p.hostSBOM {
+func (p *processor) processHostScanResult(result sbom.ScanResult) {
```

Comment:
With this new implementation the `processHostScanResult` is sequential in the main `for loop`. Is it expected? (I suppose so based on the pull request description) 
What is the trade-offs of this new design?

Reply:
Indeed it is expected. The trade-off is that the processing of host sbom and container image sbom will not parallel anymore which isn't so important given that the processing function does not have any long running operation and host sboms are rarely generated. I think we could say that this is a negligible trade off. However, we now have 1 sender goroutine and can safely close the channel.